awscli/s3transfer/crt.py

```diff
@@ -114,7 +119,12 @@ def create_s3_crt_client(
 
     :type target_throughput: Optional[int]
     :param target_throughput: Throughput target in Bytes.
-        Default is 0.625 GB/s (which translates to 5 Gb/s).
+        By default, CRT will automatically attempt to choose a target
+        throughput that matches the system's maximum network throughput.
+        Currently, if CRT is unable to determine the maximum network
+        throughput, a fallback target throughput of 1.25 GB/s
+        (which translates to 10Gb/s) is used. To set a specific target
```

Comment:
maybe spell things out? Since GB is already used meaning powers-of-2 in this codebase
```suggestion
        throughput, a fallback target throughput of 125000000 Bytes/s
        (which translates to 10 gigabits/s, or 1.16 gibibytes) is used. To set a specific target
```